app/views/content/blog/my-career-change-to-teaching.md

```diff
@@ -0,0 +1,37 @@
+---
+title: My career change to teaching
+author: Ian Wallace
+date: "2022-02-03"
+images:
+  ian:
+    path: "media/images/content/blog/ian-wallace.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/ian-wallace.jpg"
+description: |-
+  Ian Wallace had always wanted to be a teacher but didn’t feel ready when he finished university. After several different jobs and careers he finally took the plunge and hasn’t looked back.
+    
+keywords:
+  - teacher training
+  - trainee teacher course
+  - SCITT
+  - career change
+
+tags:
+  - becoming a teacher
+  - career change to teaching
+---
+
+$ian$
+
+I always wanted to be a teacher when I was at school, after the obvious football career. But when I finished university I didn’t think it was right for me. I had been in education a long time and worried I wasn’t mature enough. I went away and had several different jobs and careers at various levels of responsibility in retail, food and sports, including being self-employed a couple of times. 
+
+Time had passed and I had moved back to Newcastle, met my now wife and was contemplating my future. I was sat having a meal with my wife and she said, ‘Why don’t you teach?’ My wife is a primary school teacher and she knew that it was what I should be doing with the skill set and experience that my career had given me so far. In particular, I had skills in communication, listening, collaboration, adaptability, empathy and patience, all of which come with maturity and experience.
+
+I took the bull by the horns and looked into the qualification routes that were available, settling on a SCITT as this suited me personally. With a SCITT you are in school from the start and that appealed to my work style and ethic. Being in my late 30s and having financial responsibilities for my family I needed to have support and fortunately I was to study science, which was funded and allowed me to concentrate on my training.
```

Comment:
```suggestion
I took the bull by the horns and looked into the available qualification routes, settling on a SCITT. With a SCITT you're in school from the start and that appealed to my work style and ethic. Being in my late 30s and having financial responsibilities for my family, I needed to have support. Fortunately, I was studying science which was funded and allowed me to concentrate on my training.
```
Some quite long, complex sentences - can we rewrite these slightly for better readability?